api/team.js

```diff
@@ -1,13 +1,163 @@
 const express = require("express");
+const models = require("./models");
 const utils = require("./utils");
 const router = express.Router();
 
 router.use(utils.authMiddleware);
 router.use(utils.preprocessRequest);
 
-router.get("/", (req, res) => {
+// GET /api/team
+// - If a hacker is on a team, get that team info
+router.get("/", async (req, res) => {
+  const hackerProfile = await models.HackerProfile.findOne({
+    where: { userId: req.user.id }
+  });
+
+  const team = await hackerProfile.getTeam({
+    include: [
+      {
+        model: models.HackerProfile,
+        attributes: ["firstName", "lastName", "status", "email"]
+      }
+    ]
+  });
+
+  if (team) {
+    return res.json({ team });
+  } else {
+    return res.json({
+      message: "User does not currently belong to a team"
+    });
```

Comment:
Probably handled on the frontend, but maybe having a nullable "team"  attribute that we can check might be a bit cleaner than having this return variably shaped JSON depending on if a team exists. Alternately, 404 just so its easier to check?